### PR TITLE
Change zsh standard compatibility

### PR DESCRIPTION
**Pull request by grigorii-horos**
Function dir should be `/functions`, see: https://github.com/zdharma-continuum/Zsh-100-Commits-Club/blob/master/Zsh-Plugin-Standard.adoc#funcs-dir

### CODE REVIEW

**Comment by alichtman** [comment]
Ironically, this plug-in and the standard you linked were written by the same person :)

**Comment by alichtman** [comment]
Let's also drop all Unicode chars in file names here.

**Comment by vladdoster** [comment]
- [ ] Remove Unicode characters in file names

**Review by vladdoster — CHANGES_REQUESTED** [review]
@grigorii-horos

- [ ] Strip Unicode characters from filenames

**Comment by grigorii-horos** [comment]
@vladdoster Done!

**Review by alichtman — APPROVED** [review]
LGTM. @pschmitt why the thumbs down?

**Comment by pschmitt** [comment]
Renaming all the internal funcs is out of scope for this pr imo.

**Comment by alichtman** [comment]
While that's true, I'm ok with a "modernize this repo" PR

**Comment by alichtman** [comment]
@pschmitt Is right, now that I think about it. Let's split this into two PRs.

**Comment by alichtman** [comment]
On second thought, this PR is now impossible to review efficiently.

**Comment by alichtman** [comment]
Let's cherrypick `Remove Unicode symbol →` to a new PR and I'll approve this one.

**Comment by FranklinYu** [comment]
If we were to change the function prefix, would we also want to change

https://github.com/zdharma-continuum/Zsh-100-Commits-Club/blob/master/Zsh-Plugin-Standard.adoc#namespacing

as well? Currently it recommends `→` for hook functions. See also #4.

**Comment by grigorii-horos** [comment]
Hey @alichtman Done!

**Comment by Aloxaf** [comment]
I suggest not to remove the unicode prefix.

A unicode prefix is less likely to be a prefix of a normal function/file so that it is less likely to appear in the completion list.

The repo is deleted and I cannot find the old discussion, but the commit which change ascii prefix to unicode preifx is still here: https://github.com/zdharma-continuum/fast-syntax-highlighting/commit/a3242a93399535faccda4896ab5c61a7a6dca1fe

File: .directory [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[Dolphin]
```
Comment (vimpostor) [inline .directory]: In the first commit you deleted the Dolphin config and here you are readding it again by accident. Please remove it again from the second commit.

**Review by vladdoster — APPROVED** [review]
LGTM. Will squash the two commits on merge.

**Comment by vladdoster** [comment]
On second thought, I just saw @alichtman comment regarding splitting up changes.

@grigorii-horos, Could you please split changes into two PRs to make the changes more atomic?

**Review by vladdoster — CHANGES_REQUESTED** [review]
- [ ] PR for renaming internal functions
- [ ] PR for using `/functions`

**Pull request closed without merging**